src/main/archetype/ui.apps/pom.xml

```diff
@@ -52,6 +52,13 @@
                     <name>${artifactId}</name>
                     <packageType>application</packageType>
                     <accessControlHandling>merge</accessControlHandling>
+                    <validatorsSettings>
```

Comment:
This just hides the dirt under the rug.

Reply:
I agree, the warning states:

```
Skip validation of nodes with that type/name (org.apache.jackrabbit:filevault-package-maven-plugin:1.1.4:validate-files:default-validate-files:process-classes)
```

but I didn't find anything else to skip this check, I checked this docs:
http://jackrabbit.apache.org/filevault-package-maven-plugin/validators.html
https://jackrabbit.apache.org/filevault/validation.html

Reply:
looks like the right way is to use cnds:

https://github.com/adobe/aem-project-archetype/pull/502/files#diff-3ed42c4af924216a42188f4eb91aeceeR303-R318